docs/concepts/automations.md

```diff
@@ -96,27 +96,26 @@ from prefect.events.schemas.automations import EventTrigger
 from prefect.server.events.actions import CancelFlowRun
 
 # creating an automation
-automation =
-  Automation(
+automation = Automation(
     name="woodchonk",
     trigger=EventTrigger(
-      expect={"animal.walked"},
-      match={
-        "genus": "Marmota",
-        "species": "monax",
+        expect={"animal.walked"},
+        match={
+            "genus": "Marmota",
+            "species": "monax",
         },
         posture="Reactive",
         threshold=3,
-        ),
-      actions=[CancelFlowRun()]
```

Comment:
This previously _looks_ like action sis a part of a trigger's definition rather than being a sibling